src/components/profile-item/ProfileItem.tsx

```diff
@@ -22,9 +23,21 @@ const ProfileItem = ({
   const { t } = useTranslation()
   const { isMobile } = useBreakpoints()
   const { id, icon } = item
+  const navigate = useNavigate()
+
+  const isClickable = !isFilled && item.id !== 'schedule'
+
+  const handleItemClick = () => {
+    if (isClickable) {
+      navigate(`${item.path}#${item.id}`)
+    }
+  }
 
   return (
-    <Box sx={{ position: 'relative' }}>
+    <Box
+      onClick={handleItemClick}
+      sx={{ position: 'relative', cursor: isClickable ? 'pointer' : '' }}
```

Comment:
try to move styles to separate file

Reply:
did it! I only left the conditional style (it's normal to do in our project)